src/components/Nav/Nav.jsx

```diff
@@ -1,7 +1,55 @@
-import React from "react";
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Nav.scss';
+import '../../styles/variables.scss';
 
 const Nav = () => {
-  return <div></div>;
+  const navigate = useNavigate();
+  const goHome = () => {
+    navigate('/');
+  };
+
+  const goBooking = () => {
+    navigate('/booking');
+  };
+
+  const [scrollY, setScrollY] = useState(0);
+  const [scrollActive, setScrollActive] = useState(false);
+
+  const scrollFixed = () => {
+    if (scrollY > 100) {
+      setScrollY(window.scrollY);
+      setScrollActive(true);
+    } else {
+      setScrollY(window.scrollY);
+      setScrollActive(false);
+    }
+  };
```

Comment:
scrollY state에 의해 계산되는 값이라면, scrollActive를 state로 만들 필요 없이 변수로 사용할 수 있겠네요!
```suggestion
  const handleScroll = () => {
    setScrollY(window.scrollY);
  };

  const isNavFixed = scrollY > 100;
```

Reply:
그리고 값이 boolean이기 때문에, boolean naming convention(is / has / should / ...)에 맞춰서 네이밍했습니다!

Reply:
수정 완료되었습니다!